### PR TITLE
[CUR-692] lti navigation fixes

### PR DESCRIPTION
**Pull request by muhammadqamar**

### CODE REVIEW

File: src/containers/Preview/PlaylistPreview/components/NextLink.js [hunk 1]
```diff
@@ -77,7 +77,11 @@ function NextLink(props) {
                   })
                     .then((result) => {
                       if (result.value) {
-                        history.push(`/project/${projectId}/${shared ? 'preview' : 'preview'}`);
+                        if (showLti) {
+                          history.push(`/project/${projectId}/shared`);
+                        } else {
+                          history.push(`/project/${projectId}/${shared ? 'preview' : 'preview'}`);
```
Comment (simonebelotti75) [inline src/containers/Preview/PlaylistPreview/components/NextLink.js:83]: Ternary operator looks weird here
Is it same 'preview' whether shared is true or false?

File: src/containers/Preview/PlaylistPreview/components/PreviousLink.js [hunk 2]
```diff
@@ -76,7 +76,11 @@ function PreviousLink(props) {
                   })
                     .then((result) => {
                       if (result.value) {
-                        history.push(`/project/${projectId}/${shared ? 'preview' : 'preview'}`);
+                        if (showLti) {
+                          history.push(`/project/${projectId}/shared`);
+                        } else {
+                          history.push(`/project/${projectId}/${shared ? 'preview' : 'preview'}`);
```
Comment (simonebelotti75) [inline src/containers/Preview/PlaylistPreview/components/PreviousLink.js:82]: Same here

**Review by simonebelotti75 — CHANGES_REQUESTED** [review]

File: src/containers/Preview/PlaylistPreview/components/PreviousLink.js (hunk 2 above)
Comment (muhammadqamar) [reply src/containers/Preview/PlaylistPreview/components/PreviousLink.js]: updated

File: src/containers/Preview/PlaylistPreview/components/NextLink.js (hunk 1 above)
Comment (muhammadqamar) [reply src/containers/Preview/PlaylistPreview/components/NextLink.js]: updated

**Review by simonebelotti75 — APPROVED** [review]

**Pull request merged**